std/digest/package.d

```diff
@@ -433,10 +432,10 @@ DigestType!Hash digest(Hash, Range)(auto ref Range range)
 if (!isArray!Range
     && isDigestibleRange!Range)
 {
-    import std.algorithm.mutation : copy;
+    import std.range.primitives : put;
     Hash hash;
     hash.start();
-    copy(range, &hash);
+    put(hash, range);
```

Comment:
Can revert, and also switch to UFCS if you want.

Reply:
Since this one isn't a public example, and std.range is more likely to have already been imported, I'll leave this

Reply:
OK, I didn't notice that it wasn't an example.